contracts/LifCrowdsale.sol

```diff
@@ -11,6 +11,10 @@ contract LifCrowdsale is Ownable, Pausable {
   // The token being sold
   LifToken public token;
 
+  // start and end of the public presale
+  uint256 public startPublicPresaleBlock;
+  uint256 public endPublicPresaleBlock;
```

Comment:
I will rename them, for me there is no need to have a endBlock for private presale, whats wrong in using the public presale start block?